collage/collage.py

```diff
@@ -0,0 +1,46 @@
+import tkinter as tk
+from collage.collage_image import PILCollageImage
+
+
+class Collage(tk.Canvas):
+    def __init__(self, margin, master_args, master_kwargs):
+        super().__init__(*master_args, **master_kwargs)
+        self.margin = margin
+        # collage will be placed on the center of the widget
+        self.grid(row=1, column=1)
+        # TODO class to manage images
+        self.images = dict()
+        self.image = None
+        self.selection_rectangle_id = self.create_rectangle(
+            0, 0, 0, 0, dash=(2, 2), fill='', outline='white', tag="image"
+        )
+        self.bind("<Configure>", self.resize_event_handler)
+        self.bind("<Double-Button-1>", self.selection_area_handler)
+
+    def add_image(self, filename, where):
+        """
+        Add image in collage to specified side
+        filename: image file name
+        where: 'n', 'w', 'e', 's'
+        """
+        self.image = PILCollageImage(
```

Comment:
If we can't open image for some reason, we get _PIL.UnidentifiedImageError_ here. We should somehow inform user if this case happens.